test/toolkits/test_notion_toolkit.py

```diff
@@ -0,0 +1,120 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from unittest.mock import patch
+
+from camel.toolkits import FunctionTool, NotionToolkit
+
+
+def test_get_notion_block_text_content():
+    with patch('notion_client.Client') as mock_client:
+        # Mock the list_children method to return a sample block structure
+        mock_client.blocks.children.list.return_value = {
```

Comment:
need to add `.return_value`, same as below

```suggestion
        # Mock the list_children method to return a sample block structure
        mock_client = mock_client.return_value
        mock_client.blocks.children.list.return_value = {
```